octopoes/octopoes/models/ooi/reports.py

```diff
@@ -39,10 +39,22 @@ class Report(OOI):
 
     observed_at: datetime
     parent_report: Reference | None = ReferenceField("Report", default=None)
+    report_recipe: Reference | None = ReferenceField("ReportRecipe", default=None)
     has_parent: bool
 
     _natural_key_attrs = ["report_id"]
 
     @classmethod
     def format_reference_human_readable(cls, reference: Reference) -> str:
         return f"Report {reference.tokenized.report_id}"
+
+
+class ReportRecipe(OOI):
```

Comment:
should include name with placeholders for date

Reply:
The name belongs to the report OOI and not the recipe OOI as I can see that we can make 1 recipe for multiple reports with different names

Reply:
But you choose the name (with place holder) during the creation of the recipe, not after a run of creating the report.

Reply:
The user choses the report name at the final step of the report creation, before it creates the report OOI and at that moment we create the recipe + report and also add the recipe to the report ooi as reference. Or do you mean something different?

Reply:
For concatenated report, let's say we choose all report types, around 11 types. The final step shows 11 report names to change. we create 11 report oois with 1 recipe ooi, then we have 11 different report names that points to that 1 recipe.

Reply:
I would expect the recipe to have the name _including_ placeholders, for itself, and its child-reports.

Reply:
Jup agree, for the child reports, mostly placeholders should be used. Something like `{report_type}, for {input_ooi} on {date}` parent report could then be called something `My report on {date}`

Reply:
Yes, but this is already the case. See https://github.com/minvws/nl-kat-coordination/blob/200eed578ca6f1675f4d027d47e4f2d974ba438a/octopoes/octopoes/models/ooi/reports.py#L26